frontend/micro-ui/web/micro-ui-internals/packages/modules/Measurement/src/pages/employee/CreateMeasurement.js

```diff
@@ -72,43 +105,44 @@ const CreateMeasurement = () => {
   };
 
 
+  const onFormValueChange = (setValue, formData, formState, reset, setError, clearErrors, trigger, getValues) => {
+    // setSessionFormData({ ...sessionFormData, ...formData });
 
+    if (!_.isEqual(formData, createState)) {
+      setState({ ...formData })
+    }
+    console.log(formData, "formData")
+    console.log(createState, "createStateData")
+    // console.log(sessionFormData, "sessionFormData")
+  }
 
   if (isContractLoading || isEstimateLoading) {
     return <Loader />
   }
 
-  const estimateDetails = estimate?.estimateDetails || [];
-  // console.log(estimateDetails, "eeeeeeeeeeee")
-  const sorCategoryArray = [];
-  const nonSorCategoryArray = [];
-
-  estimateDetails.reduce((_, currentItem) => {
-    if (currentItem.category === 'SOR') {
-      sorCategoryArray.push(currentItem);
-    } else if (currentItem.category === 'NONSOR') {
-      nonSorCategoryArray.push(currentItem);
-    }
-  }, null);
 
+  console.log(createState,
+    "createStatecreateState");
 
   return (
     <FormComposerV2
-      heading={t("Measurement Book")}
-      label={t("Submit Bar")}
-      description={"Description"}
-      text={"Sample Text if required"}
+      // heading={t("Measurement Book")}
+      // label={t("Submit Bar")}
+      // description={"Description"}
+      // text={"Sample Text if required"}
       config={CreateConfig({ defaultValues: contract }).CreateConfig[0]?.form?.map((config) => {
+        console.log(config, "cccccccccccccc")
         return {
           ...config,
           body: config.body.filter((a) => !a.hideInEmployee),
         };
       })}
-      defaultValues={{ SOR: sorCategoryArray, NONSOR: nonSorCategoryArray }}
+      defaultValues={{ ...createState }}
```

Comment:
@rachana-egov  that rerender issue is fixed?

Reply:
yeah i had fixed the rendering issue on that day.

Reply:
SOR value was getting set as undefined ?
that is resolved ??